### DISCUSSION TITLE
how to add old password input when patch/update my password?

### DISCUSSION THREAD

**Discussion by fantasyhh** [fastapi-users/fastapi-users#667] [General]
when update my password ,  i want to input my old  password   to verify

**Comment by frankie567** [comment]
Hi there,

Sorry for the delayed response 🙂

Currently, this is not possible out-of-the-box. What you could do is to implement an endpoint yourself that checks the current password before proceeding. You can draw inspiration from the current update user endpoint:

https://github.com/frankie567/fastapi-users/blob/6374cf4b7a9d3d4a6e7b2d87c2c9657687701bda/fastapi_users/router/users.py#L54-L69
